src/mam4xx/mo_gas_phase_chemdr.hpp

```diff
@@ -329,6 +343,10 @@ void perform_atmospheric_chemistry_and_microphysics(
     // the logic below is taken from the aero_model_gasaerexch
     // subroutine in eam/src/chemistry/modal_aero/aero_model.F90
 
+    // Store mixing ratios before aqueous chemistry changes the mixing ratios
+    Real vmr1[gas_pcnst] = {};
+    for (int i = 0; i < gas_pcnst; ++i)
+      vmr1[i] = vmr[i];
```

Comment:
Can we change the variable name from `vmr1` to `vmr_bef_aq_chem`, since this is a new variable?